### ISSUE TITLE
Need help understanding the repo

### ISSUE THREAD

**Issue by jayavanth** [davidsandberg/facenet#409]
I'm a little confused about the pre-trained models here. As I understand it, both pre-trained models 20170511-185253 and 20170512-110547 are trained as classifiers. Is there a script that I can use like `src/classifier.py CLASSIFY` to classify some test images?
I also noticed that validate_on_lfw.py, which validates the two pre-trained models, uses the embeddings `embeddings:0`. Why not just use the last layer `softmax` to get the class.

**Comment by pavankumarkatakam** [comment]
hi @jayavanth, the both models are trained on Ms-celeb data set so if you use softmax to classify then your images to tends to show the likelihood of image in MS-Celeb dataset rather than on your own set of images. So use embeddings for your own image set and make an own classifier like SVM etc.

**Comment by jayavanth** [comment]
@pavankumarkatakam I want a classifier that is trained on Ms-Celeb. I was wondering if there was a script to do the forward prop.
